doc/parallel.rst

```diff
@@ -32,7 +32,7 @@ This backend will use MPI (Message Passing Interface) on the system to split neu
 
 **MacOS Dependencies**::
 
-    $ conda install yes openmpi mpi4py
+    $ conda install -y openmpi mpi4py
```

Comment:
the contributing guide still recommends the pip install. Perhaps we should update it and point here?

Reply:
I can't really tell why `mpi4py` is included in "Building the Documentation". It seems like it would fail on macs. Also, it may not even be necessary for just building the docs. I think the `plot_simulate_evoked.py` example will fall back to the Joblib backend.

I did add a link to this page at the top of the contribution guide.